src/services/notifications.service.js

```diff
@@ -176,7 +157,9 @@ const sendEmailOfType = async (
     );
     throw err;
   }
-  return emailSent;
+  statusEmitter.emit("incrementCount", "emailNotificationsSucceeded");
+  statusEmitter.emit("setStatus", "mostRecentEmailNotificationSucceeded", true);
+  logEmitter.emit("functionSuccess", "registration.service", "sendEmailOfType");
 };
 
 /**
```

Comment:
JsDoc not updated

Reply:
JsDoc now updated

Reply:
Function description still needs to be updated